src/client/validation.ts

```diff
@@ -81,6 +82,28 @@ export function parseAppChannelName(name: string): ChannelId {
     return name;
 }
 
+/**
+ * Validates the provided number is an integer
+ */
+export function parseInteger(number: number): number {
```

Comment:
I'd prefer to use something like `value` to resolve the ambiguity. can't tell if you mean the type name or argument name when you use `number` in the body.

Reply:
✅